src/Core/Access.php

```diff
@@ -55,14 +55,15 @@ public static function can_access($page_groups=[], $user_groups=[]) {
         return false;
     }
 
-    public static function onLoginSuccess($redirectPath) {
+    public static function onLoginSuccess($redirectPath): Message {
```

Comment:
since these methods here in access havent been in a release, can change them up at will without caring about backwards compat